custom_components/google_home/api.py

```diff
@@ -178,17 +196,7 @@ async def get_alarms_and_timers(
                 "Request error: %s",
                 ex,
             )
-        finally:
```

Comment:
I believe this `finally` is actually redundant because:

> The finally block lets you execute code, regardless of the result of the try- and except blocks.

Therefore, i have just moved the response parsing close to where `resp` is obtained